src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.byte.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace System.Memory.Tests
 {
-    public abstract class ReadOnlySequenceTestsByte
+    public abstract class ReadOnlySequenceTestsByte: ReadOnlySequenceTests<byte>
```

Comment:
It is difficult to review exactly which tests got added or changed due to all the code refactoring.

Can you please make the refactoring change as a separate commit (or even PR)?

Reply:
I made so to not make copies of the same code. I will add separated PR for tests refactiring.

Reply:
I will make so after https://github.com/dotnet/corefx/pull/29262 PR will be merged because it is required for this PR.